trussworks/react-uswds
# build(node): Upgrade to nodejs 18

# Summary

Upgrades node from 16.x to 18.x since 16 is EOL 2023-09-11 See [nodejs release schedule](https://github.com/nodejs/release#release-schedule) and [update about the EOL of node 16.x](https://nodejs.org/en/blog/announcements/nodejs16-eol/)

Also drops support for unsupported versions of node 10.x - 12.x

## Related Issues or PRs

#2129 

## How To Test

Ensure you can install dependencies, run various commands like `yarn lint`, `yarn test`, and `yarn storybook`.
